test/integration/team.js

```diff
@@ -18,22 +18,5 @@ describe('Team', () => {
         })
         .catch(done);
     });
-
-    it('select-user rpc request is successful', (done) => {
```

Comment:
Why do we want to delete this?

Reply:
Given this diff starts running integ tests on PRs, I didn't want to fumble around to get a team setup and we don't have a team that I know is ok to be using their select user functionality.  I will re-add once we setup official test apps to use here

Reply:
Gotcha. Just make sure to track it so we don't forget :)